src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -211,6 +211,37 @@ function validate_boolean_string_value(value) {
     return false;
 }
 
+/**
+ * validate_no_leading_whitespace validates any flag value does not contain leading whitespace in the command
+ *
+ * ### CLI Parsing Behavior:
+ * When passing arguments via CLI, if a flag's value starts with a space or consists only of whitespace,
+ * the shell may treat it differently:
+ *  Leading spaces - might be preserved when quoted (`--access_key=" 123abc"`).
+ *  Unquoted values with spaces - may cause incorrect parsing (`--access_key= 123abc` is interpreted as an empty value).
+ *
+ * @param {string[]} argv
+ * @throws {ManageCLIError}
+ */
+function validate_no_leading_whitespace(argv) {
+    for (let i = 0; i < argv.length; i++) {
+        if (!argv[i].startsWith('--') && !argv[i].includes('=')) continue;
+
+        const [option, value] = argv[i].split('=');
```

Comment:
this is not working correctly when value contains '=' in it

Reply:
Can you give me some example for which case it is not working. Actually, we are only evaluating argv containing '=' in it.

Reply:
just try to think about any possible value that can have = in it, here is an example - 
```
noobaa-cli account add --name=aayush====1 --uid=9 --gid=9
```
when I print option and value I get the following -
```
[LOG] CONSOLE:: ROMY option: --name
[LOG] CONSOLE:: ROMY value: aayush
[LOG] CONSOLE:: ROMY option: --uid
[LOG] CONSOLE:: ROMY value: 9
[LOG] CONSOLE:: ROMY option: --gid
[LOG] CONSOLE:: ROMY value: 9
```
value of name is incorrect

Reply:
also you can have no = between the option and the value and have it only in the value
for example - 
noobaa-cli account add --name aayush====1 --uid 9 --gid 9